### PR TITLE
Task/sanity check

### PR DESCRIPTION
**Pull request by dmoranj**
Add a new administration port with a resource /version. Address issue #164

### CODE REVIEW

**Comment by jcalderin** [comment]
LGTM

File: lib/fiware-orion-pep.js [hunk 1]
```diff
@@ -190,6 +179,86 @@ function startProxy(callback) {
...
+    } else {
+        proxyObj.adminServer = http.createServer(proxyObj.administration);
+    }
+
+    proxyObj.adminServer.listen(
+        proxyObj.administration.get('port'),
+        proxyObj.administration.get('host'), function startServer(error) {
+
+        if (error) {
+            logger.error('[PROXY-GEN-003] Error initializing administration server: ' + error.message);
```
Comment (fgalan) [inline lib/fiware-orion-pep.js]: This PROXY-GEN-003 error is documented in the corresponding .md file (I don't remember which one right now)?

File: lib/middleware/administration.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ */
+
+'use strict';
+
+var packageInformation = require('../../package.json'),
+    config = require('../../config');
+
+function retrieveVersion(req, res, next) {
+    res.status(200).json({
+        version: packageInformation.version,
```
Comment (fgalan) [inline lib/middleware/administration.js]: Do packageInformation object "maps" the information in package.json?

(Just curiosity)

Comment (dmoranj) [inline lib/middleware/administration.js]: Yes, it does.

File: lib/fiware-orion-pep.js (hunk 1 above)
Comment (dmoranj) [inline lib/fiware-orion-pep.js]: It is documented in the operations.md file, but just for the case of the proxy (is the same kind of error, so I don't think a second error type is needed). I've updated the operations.md information to add the alternative log message in a182244.

File: lib/middleware/administration.js (hunk 2 above)
Comment (fgalan) [inline lib/middleware/administration.js]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**